app/controllers/api/v1/search_controller.rb

```diff
@@ -138,14 +138,18 @@ class SearchController < ApiBaseController
 
       def index
         @viewable = Study.viewable(current_api_user)
-
+        @search_type = :keyword
```

Comment:
That's a valid point... really what this variable is tracking is how we want to sort our results for relevance.  So I'll change that to `sort_type` since this is what it is actually gating.